apis/apiextensions/v1/composition_transforms.go

```diff
@@ -27,8 +27,7 @@ import (
 
 	"github.com/crossplane/crossplane-runtime/pkg/errors"
 
-	xperrors "github.com/crossplane/crossplane/pkg/validation/errors"
-	"github.com/crossplane/crossplane/pkg/validation/schema"
+	xperrors "github.com/crossplane/crossplane/internal/errors"
```

Comment:
Nit: Maybe `verrors` or `validationerrors`? To distinguish that these errors pertain to validation, not generic "crossplane" errors as is implied by the prefix "xp".

Reply:
I went with `verrors` https://github.com/crossplane/crossplane/pull/3986/commits/62ec5ba5eaa314385734434d46e2dd94bd1515fb